packages/core/src/domain/configuration/configuration.spec.ts

```diff
@@ -136,6 +137,8 @@ describe('validateAndBuildConfiguration', () => {
 
     it('should be set to 10KB when lower-batch-size is enabled', () => {
       updateExperimentalFeatures(['lower-batch-size'])
+      setSampledExperimentalFeatures({ 'lower-batch-size': 100 })
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏this should be set before update